### ISSUE TITLE
Firefox Sync backup issues

### ISSUE THREAD

**Issue by JasonSwindle** [kickscondor/fraidycat#124]
**Before you start.**

* [x] I have checked the [Common Problems](https://github.com/kickscondor/fraidycat/wiki/Common-Problems) page.
* [x] I have searched the issue tracker for a bug that matches the one I want to file, without success.

**Desktop (please complete the following information):**
 - OS: Linux, Mac
 - Browser: Firefox 
 - Version: 68, 74+
 - URL: N\A

**Describe the bug**
Using Firefox Sync, my states of FraidyCat are all over the place. I was expecting my state (followed items) to be the same on all browsers on all devices.  I also made sure that I Firefox Sync was synced.

**Comment by egrajeda** [comment]
:wave: Hey there! I've also been having this exact same problem (or a **pretty** similar one) since my follows started to grow, so I spent some time chasing the cause and working on a fix.

I submitted a PR in case you want to consider it (#143) and I summarized what I discovered there.

**TL;DR**: Firefox doesn't enforce the 8kb per-item sync limit when storing it, just when it syncs it to the server in the background. Because of this, the already built-in logic in `frago.js` to partition your follows data doesn't kick-in on Firefox, just in Chrome.

**Comment by kickscondor** [comment]
Ok, wow - thank you, Eduardo! I hadn't made time to track this down - but it was perplexing me because Chrome-based browsers have been so solid at syncing.

I haven't done the testing and auditing necessary to merge this change - but I'm very grateful for the time you put into this. It's an immense help when someone takes initiative like this. So I will be glad to put in the time to getting this PR merged.

So yeah - fantastic!
